inst/quarto_website/quarto_website.yml

```diff
@@ -19,7 +19,7 @@ website:
     collapse-level: 1
     contents:
       - text: Home
-        file: index.qmd
```

Comment:
I think we should encourage `.qmd` by default, since `.md` makes some Quarto features unavailable.

Reply:
> since `.md` makes some Quarto features unavailable.

For example?

Reply:
I think I got confused with the order of operations. `index.md` is fine here because it will link to the `README.md` that we copied over, or to the `README.qmd` after it was rendered to `.md`. Sorry for the confusion.